cmd/gossamer/configcmd.go

```diff
@@ -328,3 +338,24 @@ var tomlSettings = toml.Config{
 		return fmt.Errorf("field '%s' is not defined in %s%s", field, rt.String(), link)
 	},
 }
+
+// tilde will expand a tilde prefix path to full home path
+func tilde(targetPath string) string {
```

Comment:
I think something like `expandTilde` would be more descriptive

Reply:
sure